ui/src/app/components/common-env-conf-panels/environment-cost-estimator.tsx

```diff
@@ -135,7 +135,9 @@ export const EnvironmentCostEstimator = ({
           </div>
         )}
       </FlexColumn>
-      {!!pdCost && (
+      {/* Jupyter doesn't display the Persistent Disk cost when the disk size is zero. For GKE, Since we only show the
```

Comment:
I don't understand this comment. could you clarify?

Reply:
For jupyter we display Cost when running, cost when paused and PD cost. In case user selects Disk size as 0 (temp) PD cost disappers.

https://github.com/all-of-us/workbench/assets/34481816/3736ce55-c5fe-40aa-a75e-c1a0deb1846f


SInce GKE Apps just display Cost when running and PD cost, removing PD Cost sections causes more distraction

SO I Kept the functionality as is for Jupyter but in case OF apps if PD cost is 0 still show the PD COST

Reply:
This video is not working for me unfortunately.

I think it would make the logic clearer here to use a variable `showPdCost` and leave a simple comment where you define it like "always show for GKE, but only show for Jupyter when a PD exists"

Reply:
I will update it , hopefully there are no question abut why difference of behavior  later